src/dev/konnector.md

```diff
@@ -244,7 +257,7 @@ If you get this problem, catch the error yourself and only display the message :
 }
 ```
 
-### How do I scrap a website
+### <a name="How-do-I-scrape-my-data-from-a-website"></a>How do I scrap a website
```

Comment:
For the record, you don't have to manually add anchors, mkdocs create them automatically (drag your mouse pointer hover any subtitle and you'll see its direct link at the end of the line).